routes/login.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
```

Comment:
We need to decide whether we are using beef.js or index.js but definitely not both.